app/src/main/java/com/sunasterisk/fooddaily/ui/splash/SplashScreenContracts.kt

```diff
@@ -0,0 +1,15 @@
+package com.sunasterisk.fooddaily.ui.splash
+
+import com.sunasterisk.fooddaily.data.model.FoodDetail
+import kotlin.Exception
+
+interface SplashScreenContracts {
+    interface View {
+        fun onDataLoadedSuccess(data: List<FoodDetail>)
+        fun onDataLoadedFailure(exception: Exception)
```

Comment:
Đặt tên rõ ràng, muốn làm gì với dữ liệu kia với View 

```suggestion
        fun showFoods(data: List<FoodDetail>)
```